pyogrio/tests/test_geopandas_io.py

```diff
@@ -187,40 +190,105 @@ def test_read_fids_force_2d(test_fgdb_vsi):
 
 
 @pytest.mark.parametrize(
-    "driver,ext",
-    [
-        ("ESRI Shapefile", "shp"),
-        ("GeoJSON", "geojson"),
-        ("GeoJSONSeq", "geojsons"),
-        ("GPKG", "gpkg"),
-    ],
+        "suffix",
```

Comment:
I personally preferred `ext` (extension) over `suffix` since it was more specific, but I don't have a terribly strong opinion about it.

Reply:
No problem, it's just out of habit because I always use pathlib, and there it is called suffix.

Seems like one of those many things that don't matter, but should be consistent in a code base... so I'll fix it.